apps/condo/bin/notification/send-billing-receipt-push-residents.js

```diff
@@ -1,25 +1,33 @@
 const conf = require('@core/config')
-const { flatten, uniq } = require('lodash')
+const { flatten, uniq, get } = require('lodash')
+
+const { getUniqueByField, getConnectionsMapping, getStartDates } = require('@condo/domains/common/utils/task/helpers')
 
 const { BillingReceipt, BillingProperty } = require('@condo/domains/billing/utils/serverSchema')
 
 const { Message, Device } = require('@condo/domains/notification/utils/serverSchema')
 const {
     BILLING_RECEIPT_AVAILABLE_TYPE,
     BILLING_RECEIPT_AVAILABLE_NO_ACCOUNT_TYPE,
-    MESSAGE_SENT_STATUS, MESSAGE_DELIVERED_STATUS, MESSAGE_READ_STATUS,
+    MESSAGE_SENT_STATUS, MESSAGE_DELIVERED_STATUS, MESSAGE_READ_STATUS, MESSAGE_ERROR_STATUS,
 } = require('@condo/domains/notification/constants/constants')
 
 const { Property } = require('@condo/domains/property/utils/serverSchema')
-
 const { Resident } = require('@condo/domains/resident/utils/serverSchema')
 
-const { getUniqueByField, getConnectionsMapping, getStartDates } = require('./lib/helpers')
-const { BillingContextScriptCore, prepareAndProceed } = require('./lib/billing-context-script-core')
+const { BillingContextScriptCore, prepareAndProceed } = require('../lib/billing-context-script-core')
 
 // These are needed temporarily for backwards compatibility in order not to add extra migrations
 const BILLING_RECEIPT_AVAILABLE_MANUAL_TYPE = 'BILLING_RECEIPT_AVAILABLE_MANUAL'
 const BILLING_RECEIPT_AVAILABLE_NO_ACCOUNT_MANUAL_TYPE = 'BILLING_RECEIPT_AVAILABLE_NO_ACCOUNT_MANUAL'
+const SKIP_ADDRESSES = {
+    '1c22760e-b574-4115-a794-54eabba6f0a5': 'г Москва, ул Дубнинская, д 40А к 2',
+    '5cf4e09f-cc78-47d7-9da1-9db0ca574b6e': 'г Москва, ул Дубнинская, д 40 к 2',
+    '61c5cddf-1d15-49a4-be9a-021b6a2c8c79': 'г Москва, ул Дубнинская, д 40А к 1',
+    '6404b72e-1abb-4afa-9f02-1c919c6980c3': 'г Москва, ул Дубнинская, д 40А к 3',
+    'c51ee464-f4c6-4872-8b2b-9170e98d678a': 'г Москва, ул Дубнинская, д 40А к 4',
+}
+const SKIP_ADDRESS_IDS = Object.keys(SKIP_ADDRESSES)
```

Comment:
It seems like it would be good to make this into db or .env

Reply:
removed, that was temporary thing.